src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -397,6 +397,19 @@ public virtual FieldSymbol TupleUnderlyingField
             }
         }
 
+        /// <summary>
+        /// If this is a field of a tuple type,
+        /// id is an index of the element (zero-based).
```

Comment:
Also, should probably say "If this is a field representing a tuple element", rather than "If this is a field of a tuple type", because some fields of a tuple type return negative numbers.

---
In reply to: [64289994](https://github.com/dotnet/roslyn/pull/11457#discussion_r64289994) [](ancestors = 64289994)